packs/slack/sensors/slack_sensor.py

```diff
@@ -180,6 +180,7 @@ def _handle_message(self, data):
                 'is_group': channel_info.get('is_group', False),
             },
             'timestamp': int(float(data['ts'])),
+            'timestamp_raw': data['ts'],
```

Comment:
Yeah, adding additional attribute is a better since it's good to still preserve timestamp as an int so user perform different comparison operations on it.